x/confidentialtransfers/client/cli/tx.go

```diff
@@ -173,3 +168,285 @@ func makeCloseAccountCmd(cmd *cobra.Command, args []string) error {
 
 	return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
 }
+
+// NewTransferTxCmd returns a CLI command handler for creating a MsgTransfer transaction.
+func NewTransferTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "transfer [denom] [to_address] [amount] [flags]",
+		Short: "Make a confidential transfer to another address",
+		Long: `Transfer command create a confidential transfer of the specified amount of the specified denomination to the specified address. 
```

Comment:
nit: We could also add some example commands in long description. I think i rushed my piece in without it though